src/locales/en.json

```diff
@@ -89,8 +89,22 @@
     "creation_error": "An error occured while trying to contact the other app. Please try again later?"
   },
   "manifest": {
-    "short_description": "Cozy Contacts helps you to save and secure your contacts on your Cozy.",
-    "long_description": "With Cozy Contacts you can:\n- Add, modify and sort your contacts.\n- Imports all your contacts from Google, Facebook and more."
+    "short_description": "Cozy Contacts helps you to save, secure and manage all your contacts from your Cozy.",
+    "long_description": "With Cozy Contacts you can:\n\n- Add, edit and sort your contacts,\n* Manage your groups of contacts,\n* Make your sharing easier from Cozy Drive with your contacts,\n* Synchronise your contacts from Google,\n* And soon, synchronise your contacts from your mobile and other online services (iCloud, Facebook...)",
+    "permissions": {
+      "apps": {
+        "description": "Required by the cozy-bar to display the icons of the apps"
```

Comment:
should we really talk about `cozy-bar` here? See https://trello.com/c/zydzRlAi/1678-%F0%9D%90%97%F0%9D%90%92-app-contacts-manifest-wording#comment-5c867cc91d81322b6008e839